src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -132,6 +132,7 @@ internal static class IDEDiagnosticIds
 
         public const string InvalidSuppressMessageAttributeDiagnosticId = "IDE0076";
         public const string LegacyFormatSuppressMessageAttributeDiagnosticId = "IDE0077";
+        public const string RemoveUnnecessarySuppressionDiagnosticId = "IDE0078";
```

Comment:
i'd love a dedicated ID for this since it's a meta-diagnostic-id.

Reply:
Also, this id clashes with https://github.com/dotnet/roslyn/pull/44824

Reply:
@Youssef1313 would you minding using the next ID, so all the 3 unnecessary suppression IDs can be consecutive here (IDE0076, IDE0077 and IDE0078)?